kie-wb-common-services/kie-wb-common-datamodel/kie-wb-common-datamodel-backend/src/test/java/org/kie/workbench/common/services/datamodel/backend/server/ModuleDataModelFactAnnotationsTest.java

```diff
@@ -120,6 +121,7 @@ public void testModuleDMOAnnotationAttributes2() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
please add to ignore also a jira for resolving these failures later.
So create a AF jira and link it here

Reply:
@Ignore ticket https://issues.redhat.com/browse/RHPAM-4546

Reply:
I meant this exactly ;)

```suggestion
    @Ignore("https://issues.redhat.com/browse/RHPAM-4546")
```